sdk/storage/azure_storage_blob/src/generated/clients/blob_client.rs

```diff
@@ -102,6 +102,35 @@ impl BlobClient {
         })
     }
 
+    pub fn new_no_credential(
```

Comment:
GEN CODE SUPPORT: Credential-less `new()` that also doesn't call `endpoint.set_query(None)` and clobbers all query params.